content/news/2024/2024-06-20-performance-portability-training-series.md

```diff
@@ -0,0 +1,18 @@
+---
+contributor: max
+date: '2024-06-20T14:10:22.153253'
+external_url: https://www.alcf.anl.gov/events/portable-sycl-code-using-onemkl-amd-intel-and-nvidia-gpus
+title: 'Portable SYCL Code Using oneMKL on AMD, Intel, and Nvidia GPUs'
+image: ../../../static/images/news/2024-06-20 performance-portability-series.webp
```

Comment:
There is a space between '2024-06-20' and 'performance-portability-series.webp', should this be a '-'?

Reply:
I'll fix this.